homeassistant/components/fully_kiosk/icons.json

```diff
@@ -0,0 +1,7 @@
+{
+  "services": {
+    "load_url": "mdi:link",
+    "set_config": "mdi:cog",
+    "start_application": "mdi:power"
```

Comment:
`mdi:power` makes me feel like it controls some kind of power.. I wonder if something like `mdi:exit-to-app` or `mdi:rocket-launch` would be better since this service is launching a new app on the tablet and sending Fully to the background

Reply:
```suggestion
    "start_application": "mdi:rocket-launch"
```